### ISSUE TITLE
Remove numpy data downloaded by the test

### ISSUE THREAD

**Issue by sandrich** [sandrich/classifying_digits_mnist#5]
It seems that numpy datasets are uploaded into git after having run the test locally. We should not include them since we pull it from the internet

**Comment by bigskapinsky** [comment]
Thing is, all the ext_***.py are the expected values from the tests, so they are really crucial to the tests running smoothly.

**Comment by sandrich** [comment]
Nothing wrong if the test is downloading them but we should not include them in git with the exception of the files required to test against

'''tests/exp_*'''

I have removed the '''/*.py''' files which are the downloaded datasets from the internet but kept the ones in the tests subfolder

**Comment by bigskapinsky** [comment]
Ok. It's weird though, I specifically added a teardown for each test to remove any files it downloaded:
```python
   # cleanup any downloaded files
    os.unlink(f"{which}_data.npy")
    os.unlink(f"{which}_labels.npy")
    os.unlink(f"{which}_dims.npy")
```

**Comment by sandrich** [comment]
@bigskapinsky I think that works well. I added also *.npy into gitignore but forced the numpy test datasets into git since they are needed for the tests

**Comment by sandrich** [comment]
This should be fixed with #7
